python/src/aac/validation/ValidatorPlugin.py

```diff
@@ -0,0 +1,37 @@
+"""ValidatorPlugin class."""
+from __future__ import annotations
```

Comment:
Can we delete this? It doesn't seem to be used anywhere.

```suggestion
```

Reply:
The future annotation is necessary for the type hinting on line 24.